### PR TITLE
Initial infrastructure - creates 2 linux AWS instances

### PR DESCRIPTION
**Pull request by adrianz-kainos**

### CODE REVIEW

File: Infrastructure/providers.tf [hunk 1]
```diff
@@ -0,0 +1,36 @@
+provider "aws" {
+	region = "${var.region}"
+}
+
+resource "aws_instance" "msvc_1" {
```
Comment (xjedam) [inline Infrastructure/providers.tf]: could we extract creating EC2 instances to different files, so we have a clear view on what is defined where just by looking at the file listing?

File: Infrastructure/variables.tf [hunk 2]
```diff
@@ -0,0 +1,11 @@
+variable "region" {
+    default = "eu-west-1"
+}
+
+variable "amis" {
+    default = {
+        eu-west-1 = "ami-f9dd458a" 	# Amazon Linux AMI HVM EBS-Backed 64 bit
+        # eu-west-1 = "ami-75197f06" 	# Demo Microservices based on Amazon Linux AMI
```
Comment (xjedam) [inline Infrastructure/variables.tf]: do we need this?

File: Infrastructure/providers.tf (hunk 1 above)
Comment (adrianz-kainos) [inline Infrastructure/providers.tf]: Done.

File: Infrastructure/variables.tf (hunk 2 above)
Comment (adrianz-kainos) [inline Infrastructure/variables.tf]: Nope. Done.

**Comment by xjedam** [comment]
looks good

**Pull request merged**